sw/device/silicon_creator/manuf/data/manuf_testplan.hjson

```diff
@@ -0,0 +1,220 @@
+            - Verify that OTP write was successful.
+            '''
+      tags: ["fpga", "silicon"]
+      stage: V2
+      tests: []
+    }
+
+    {
+      name: manuf_cp_device_info_flash_wr
+      desc: '''Verify that the tester is able to write to flash info pages.
+
+            During early manufacturing stages, the tester writes information for later use into
+            flash info pages.
+
+            - Pre-load OTP with TEST_UNLOCKED lc_state in pre-silicon environments. Advance to
+              state in silicon.
+            - Select CPU JTAG via strap pins.
+            - Load program into SRAM to execute flash write sequences.
+            - Write 32B to the isolated flash info partition. Verify unable to read back.
+            - Write 64B to the flash secret partition. Verify result.
+
+            In silicon, a separate test program must perform the following additional operations.
+            In pre-silicon, they can be part of the same program.
+
+            - Switch to LC TAP at runtime.
+            - Request transition to DEV or PROD state (test both cases).
+            - Verify isolated flash info partition expected data.
+            '''
+      tags: ["fpga", "silicon"]
+      stage: V2
+      tests: []
+    }
+
+    {
+      name: manuf_cp_test_lock
+      desc: '''Verify provisioning and use of unlock tokens.
+
+
+
+            '''
+      tags: []
+      stage: V2
+      tests: []
+    }
+
+    // TODO: Move the following comment into manufacturing tests documentation.
+    // In post-silicon, all manuf_ft* test cases can be combined into a single program which
+    // can then be loaded via bootstrap mechanism. All test cases are written with the assumption
+    // that ROM execution is enabled. In pre-silicon environments, the tests can be run as
+    // separate programs and without requiring the production ROM.
+    {
+      name: manuf_ft_exit_token
+      desc: '''
+
+            '''
+      tags: []
+      stage: V2
+      tests: []
+    }
+
+    {
+      name: manuf_ft_sku_individualization
+      desc: '''
+
+            '''
+      tags: []
+      stage: V2
+      tests: []
+    }
+
+    {
+      name: manuf_ft_provision_rma_token
+      desc: '''
+
+            '''
+      tags: []
+      stage: V2
+      tests: []
+    }
+
+    {
+      name: manuf_ft_personalization
+      desc: '''
+
+            '''
+      tags: []
+      stage: V2
+      tests: []
+    }
+
+    {
+      name: manuf_ft_load_transport_image
+      desc: '''
+
+            '''
+      tags: []
+      stage: V2
+      tests: []
+    }
+
+    {
+      name: manuf_ft_load_certificates
+      desc: '''
+
+            '''
+      tags: []
+      stage: V2
+      tests: []
+    }
+
+    {
+      name: manuf_ft_eom
+      desc: '''
+
+            '''
+      tags: []
+      stage: V2
+      tests: []
+    }
+
+    {
+      name: manuf_rma_entry
+      desc: '''
+
+            '''
+      tags: []
+      stage: V2
+      tests: []
+    }
+  ]
```

Comment:
this part is still kind of on us because we never updated the scripts..but we should have a test that also loads "scramble" key for the first time and the system can actually transition to a new flash scramble key without shooting itself in the foot.